x-pack/legacy/plugins/apm/public/new-platform/plugin.tsx

```diff
@@ -51,11 +52,14 @@ const App = () => {
     <MainContainer data-test-subj="apmMainContainer">
       <UpdateBreadcrumbs routes={routes} />
       <Route component={ScrollToTopOnPathChange} />
-      <Switch>
-        {routes.map((route, i) => (
-          <ApmRoute key={i} {...route} />
-        ))}
-      </Switch>
+      {/* Check if user has permission to read from apm index. */}
+      <Permission>
```

Comment:
The Permission component was added here, as it should show part of the original page, such as the Breadcrumbs